src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -211,13 +225,18 @@ const MessageContentWithContext = <
     reactionPickerVisible,
     reactionsEnabled = true,
     repliesEnabled = true,
-    retrySendMessage,
     setActionSheetVisible,
     showActionSheet,
     supportedReactions = emojiData,
     threadList,
   } = props;
 
+  const { disabled } = useChannelContext<At, Ch, Co, Ev, Me, Re, Us>();
+  const {
+    Attachment = DefaultAttachment,
+    Message,
+    retrySendMessage,
+  } = useMessagesContext<At, Ch, Co, Ev, Me, Re, Us>();
```

Comment:
for this should we do what we have done in a few places and allow these to be passed in from props but if they don't exist there then using the context value?

Reply:
we could, but Attachment isn't currently a prop on MessageList, so we'd need to add it there to pass to Message. currently we have it as a prop on Channel so it's shoved into context

Reply:
or i guess we could pull it from context in MessageList and pass into Message, lotta ways to do it i guess